### PR TITLE
Edits over: Update guideline for "=>" now that it works for void

### PR DESCRIPTION
**Pull request by chalin**
This PR implements the code changes suggested in the review comments of #766

### CODE REVIEW

**Comment by kwalrath** [comment]
I had a couple of comments on Bob's PR that aren't reflected here (but I'm not sure how Bob feels about them).

**Comment by chalin** [comment]
Right, I haven't made those changes yet. Would you like for me to do so?

**Comment by chalin** [comment]
Changes have been made on #766.

File: src/_guides/language/effective-dart/usage.md [hunk 1]
```diff
@@ -688,27 +688,64 @@ constructor, you may need to do the "private field, public getter" pattern, but
...
+{:.bad-style}
+<?code-excerpt "misc/lib/effective_dart/usage_bad.dart (arrow-long)"?>
+{% prettify dart %}
+Treasure openChest(Chest chest, Point where) =>
+    _opened.containsKey(chest) ? null : _opened[chest] = new Treasure(where)
+      ..addAll(chest.contents);
+{% endprettify %}
+
+You can also use `=>` on void members that don't return a value. This is
+idiomatic for small setters that have a corresponding getter also using `=>` and
```
[the remark this answers is not in the document]
Comment (munificent) [reply src/_guides/language/effective-dart/usage.md]: > (Maybe we should just get better syntax for setter/getter pairs, so the problem becomes moot).

**YES PLEASE**

**Pull request closed without merging**